projects/plugins/boost/app/assets/src/js/features/performance-history/graph-component/graph-component.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+	BoostScoreGraph,
+	Button,
+	Gridicon,
+	Popover,
+	Spinner,
+} from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import styles from './graph-component.module.scss';
+
+const DummyGraph = ( { children } ) => {
+	return (
+		<div className={ styles.dummy }>
+			{ children }
+
+			<BoostScoreGraph isPlaceholder={ true } />
+		</div>
+	);
+};
+
+const GraphComponent = ( {
+	periods = [],
+	startDate = 0,
+	endDate = 0,
+	needsUpgrade,
+	handleUpgrade,
+	isFreshStart,
+	handleDismissFreshStart,
+	isLoading,
+} ) => {
```

Comment:
![image](https://github.com/Automattic/jetpack/assets/1369626/77c212ac-66a4-4c63-85f3-6f17c4d47647)

IMHO we should be using TypeScript typing on the properties of our components. That way when we use the components, our editor can warn us if we're mis-using them. 

A pattern I like is a `type Props = {}` at the top of the file which documents the props that are used in this component, which is then referenced in the function definition.